test/controllers/beneficial.owner.individual.controller.spec.ts

```diff
@@ -178,7 +224,7 @@ describe("BENEFICIAL OWNER INDIVIDUAL controller", () => {
     });
 
     test(`redirect to the ${BENEFICIAL_OWNER_TYPE_PAGE} page after a successful post from ${BENEFICIAL_OWNER_INDIVIDUAL_PAGE} page with service address data`, async () => {
-      mockPrepareData.mockImplementationOnce( () => BENEFICIAL_OWNER_INDIVIDUAL_OBJECT_MOCK_WITH_SERVICE_ADDRESS_NO );
+      mockPrepareData.mockReturnValueOnce( { ...BENEFICIAL_OWNER_INDIVIDUAL_OBJECT_MOCK_WITH_SERVICE_ADDRESS_NO } );
```

Comment:
One of the new 'with params' tests was passing when run on its own but failed when run with other tests. I narrowed down the issue to the mock data being returned by the mockPrepareData being the same in the test that failed as some others and one of the other tests was modifying the mock object.
To fix it I changed all places that returned this mock data to return a clone of the object instead, so any changes made to that object would not be carried into other tests.
The clone is done using the spread syntax to copy the fields into the inside of a new object - eg
{ ...myObject }